tajo-core/src/main/java/org/apache/tajo/master/QueryInProgress.java

```diff
@@ -156,8 +161,15 @@ public boolean startQueryMaster() {
   private void connectQueryMaster() throws Exception {
     InetSocketAddress addr = NetUtils.createSocketAddr(queryInfo.getQueryMasterHost(), queryInfo.getQueryMasterPort());
     LOG.info("Connect to QueryMaster:" + addr);
-    queryMasterRpc =
-        RpcClientManager.getInstance().getClient(addr, QueryMasterProtocol.class, true);
+
+    /* A large scale cluster can be connection refused, if it reuse  */
```

Comment:
It would be better if it should be changed to 'Connection can be refused in large scale clusters if 'reuse' option is TRUE'.

In addition, this description should be also described in ```QUERY_MASTER_RPC_CLIENT_REUSE``` in TajoConf and user guide.